atmospheric_explorer/ui/Home.py

```diff
@@ -23,12 +27,23 @@
 logger.info("Checking session state")
 progress_bar.progress(0.3, "Building selectors")
 with st.form("selection"):
-    st.session_state[GeneralSessionStateKeys.SELECTED_COUNTRIES] = st.multiselect(
-        "Countries",
-        options=shapefile_dataframe()["ADMIN"].to_list(),
-        default=st.session_state.get(GeneralSessionStateKeys.SELECTED_COUNTRIES),
+    st.session_state[GeneralSessionStateKeys.SELECT_COUNTRIES] = st.checkbox(
```

Comment:
I will try this out as well as soon as possible. In the meantime, I guess the design is like this: if the user checks the "Select countries" checkbox, they can select countries with the previously existing UI. Otherwise, they can draw on the map and select any random shape. Is this correct?
There is no way to mix both things, i.e. selecting some countries + a random shape, or is there?

Reply:
Yes, the only differences are as follows:
- The user can draw a generic shape while `Select countries` is ON, but it will result on all countries touching the shape being selected
- If the use switches on the countries while a generic shape is selected, countries touching it are selected

Reply:
I don't  think there's a good way of selecting countries and a random shape, is that something we can expect them to need?

Reply:
I don't think it's super important. I remember there was something similar in the Wildfire Emission Explorer, but I'd say we can definitely keep it as it is.